app/src/main/java/swati4star/createpdf/util/ImageUtils.java

```diff
@@ -241,6 +247,17 @@ public static String saveImage(String filename, Bitmap finalBitmap) {
         return myDir + "/" + fileName;
     }
 
+    public static void selectImages(Fragment frag, int requestCode) {
```

Comment:
Add javadoc style comments to newly added functions

Reply:
This one also left.

I'll review and leave comments

Reply:
This is already done. it looks like there is a Github bug because of which its not being displayed in the PR comment page. When you click on "Files Changed" and search for this function, there is the Javadoc comment